src/main/scala/com/sksamuel/scapegoat/inspections/VariableShadowing.scala

```diff
@@ -5,7 +5,12 @@ import com.sksamuel.scapegoat.{Inspection, InspectionContext, Inspector, Levels}
 import scala.collection.mutable
 import scala.collection.mutable.ListBuffer
 
-class VariableShadowing extends Inspection("Variable shadowing", Levels.Warning) {
+class VariableShadowing extends Inspection(
+  text = "Variable shadowing",
+  defaultLevel = Levels.Warning,
+  description = "Checks for multiple uses of the variable name in nested scopes.",
+  explanation = "Variable shadowing is very useful, but can easily lead to nasty bugs in your code. Shadowed variables can be potentially confusing to other maintainers when the same name is adopted to have a new meaning in a nested scope."
```

Comment:
Hmm, this is surprising:
> Variable shadowing is very useful

Reply:
Obviously this is quite subjective like a lot of other inspections. Some languages disallow variable shadowing, some allow it in some form. I personally find it useful in small and isolated scopes where otherwise I would have to make up an arbitrary name because another identifier in the scope is already using it.